File: tests/conformance/fixtures/python/22-review-node.py
from sykli import Pipeline

p = Pipeline()

p.task("test").run("go test ./...")
p.review("review-code") \
    .primitive("lint") \
    .agent("claude") \
    .context("src/**/*.go") \
    .after("test") \
    .deterministic(True)

p.emit()

File: tests/conformance/fixtures/rust/22-review-node.rs
use sykli::Pipeline;

fn main() {
    let mut p = Pipeline::new();

    p.task("test").run("go test ./...");
    p.review("review-code")
        .primitive("lint")
        .agent("claude")
        .context(&["src/**/*.go"])
        .after(&["test"])
        .deterministic(true);

    p.emit();
}
